### PR TITLE
I18n lazy lookup, closes #272 

### LINKED ISSUE #272: i18n path for cells
Hello!
I'm using cells for some time and i'm pretty glad with how it works. However, one thing bothers me - i'm not able to use shorter version of my i18n keys when translating a text inside my cells. It's actually pretty obvious because these shorter forms are meant to be used with regular controller/view/partial entities but i was thinking - wouldn't that be lovely to make it able to use it with cells using some generic scheme? For example, if user uses `t('.label')` inside a `show.erb` cell view of `UserWidgetCell`, it should display a translation from `en.cells.user_widget.show.label`.

That would be amazing to be able to use `.label` instead of that whole chain, just like in regular rails views. I would love to make a PR but i have searched everything and i'm not able to figure out how to configure such thing in rails.

### PR DESCRIPTION
**Pull request by mbajur**
Related to: https://github.com/apotonick/cells/issues/272

Ok, here is the raw module file. But still, after hundreds of attempts, i'm still not able to make it work as it should - using `t('.something')` is not being overwriten and produces an error. It would be great if you could advice me on how that module should be actually used with cells. Included by default? Included separately for each cell?

Sorry but i just think i'm missing something simple in here; no matter what i try, my overwrite is not visible from a single cell perspective and results in error

### CODE REVIEW

**Comment by mbajur** [comment]
To be more clear - adding a private `t` method to my cell class works fine but adding it anywhere in `Cell::ViewModel` don't

**Comment by apotonick** [comment]
Ok - that's because the `I18N` module might be included _after_ you defined your method and thus overwrites your beautiful method! I'll go through this soon!

**Comment by apotonick** [comment]
@mbajur Can you add a test for that in a separate `i18n_test.rb` file? You can look up how to write such a test here: https://github.com/apotonick/cells/blob/master/test/url_helper_test.rb#L58

File: lib/cell/i18n.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+module Cell::I18n
+
+  def t(*args)
+    options   = args.last.is_a?(Hash) ? args.pop.dup : {}
+    key       = args.shift
+    cell_name = self.class.to_s.delete('Cell').underscore
```
Comment (mherold) [inline lib/cell/i18n.rb]: This is deleting too much from the name (e.g. `'MenuCell'.delete('Cell') # => 'Mnu')` ... I'd suggest e.g. `gsub(/Cell\Z/, '')` instead.

Comment (mbajur) [inline lib/cell/i18n.rb]: yeah, that's exactly what i've done locally but forgot to send a PR for that. thanks

**Comment by mbajur** [comment]
Yeah, i'd love to but i don't have much time for that right now. But i'll do that for sure in upcoming days.

**Comment by svoop** [comment]
Great addition, too bad it stalled. :cry:

**Pull request closed without merging**